drivers/openstack/client.go

```diff
@@ -437,7 +437,7 @@ func (c *GenericClient) Authenticate(d *Driver) error {
 		return err
 	}
 
-	provider.UserAgent.Prepend(fmt.Sprintf("docker-machine/v%s", version.VERSION))
+	provider.UserAgent.Prepend(fmt.Sprintf("docker-machine/v%s", version.Version))
```

Comment:
I also changed the name of these versions variables because ALL_CAPS is not idiomatic to Go, even for constants.